avocado_vt/test.py

```diff
@@ -60,6 +60,9 @@
 from autotest.client.shared import error
```

Comment:
You may take this opportunity to remove dependency for  from autotest.client.shared import error